README.md

```diff
@@ -176,7 +176,13 @@ Following are the steps to see metrics on grafana dashboard.
 ![Grafana Dashboard](images/grafana1.png)
 
 * Add Prometheus server URL.
-![Add datasource](images/add-datasource.png)
+	* Using Direct Connection: In this approach, you need to port forward the prometheus server to port 9090.
+		```
+			kubectl port-forward pod/<prometheus server-pod-name>  9090:9090
```

Comment:
new standard is to put format type when showing commands or code. So, in this case you would:
```bash
kubectl ...
```